### ISSUE TITLE
add conditional logit model

### ISSUE THREAD

**Issue by josef-pkt** [statsmodels/statsmodels#941]
**Description**

In contrast to MNLogit, here the variables differ by choice, but the parameters are the same across choices (at least for some .

special case: rectangular structure: all choices have the same variables, but different values

**Interface**

???
see also issue #940 for data handling in general multi-equation models

**Comment by jbrockmendel** [comment]
@kshedden does the existing implementation of `conditional_models` close this?

**Comment by josef-pkt** [comment]
I don't think so, this has questions related to discrete choice which is and old GSOC PR
